### DISCUSSION TITLE
Compatibility with Woo Pricing & Discounts plugin.

### DISCUSSION THREAD

**Discussion by Shazgu** [woocommerce/woocommerce#45087] [WooCommerce, Blocks, and Site Editing]
Scenario: Discounted product. Discount can only be applied to 1 unit. User has added more than 1 unit to cart.

Desired behaviour: Separate prices should be displayed for the discounted units and the full price units, as in the non-blocks cart page and the mini-basket (see screenshots).

The discounts have been added via the Woo Pricing & Discounts plugin, but I expect the behaviour would be the same regardless of how the discount is added. We have checked all other plugin and can confirm that it only happens when the blocks plugin is activated.

<img width="1680" alt="Original-cart-page-displayed-correctly" src="https://user-images.githubusercontent.com/71322363/115280992-6f2b0000-a140-11eb-9778-5c55fee3c3ae.png">
<img width="368" alt="Mini-basket-displayed-correctly" src="https://user-images.githubusercontent.com/71322363/115281011-74884a80-a140-11eb-95c5-813e894ab784.png">
<img width="1680" alt="Blocks-plugin-averaging" src="https://user-images.githubusercontent.com/71322363/115281025-794cfe80-a140-11eb-8ac5-56d43bb4eb1c.png">

**Comment by senadir** [comment]
Hey @Shazgu 

Because this is a feature coming directly from Woo Pricing & Discounts and not native to WooCommerce Core, they have to provide integrate the plugin directly with Cart and Checkout block for it to be shown like this.

WooCommerce Cart and Checkout blocks are new and are still in early access, so we're aware that some extensions might not work.

Once that is done we’ll have docs prepared and will be reaching out to Woo Extension developers to help them with getting their extensions integrated. You can also ask the plugin developer to keep an eye on those blocks so they can integrate with them once integration points are public.

We make note of all feedback we receive and we also close issues with the feedback as active issues are used for bugs or feature/enhancements. It does not mean that we don’t note or review the feedback. Closed issues can also still be commented on.

**Comment by jongore** [comment]
Thank you so much for sharing this. Actually I am also working on the same business model. Here we are offering cart discount plugins for WooCommerce stores. You can offer discount on your desired products, on returning products to reduce loss of returning items. Our plugin enables you to send an email to those customers who added the product in cart but did not purchase. You may send a follow up email to those customers in the form of a discount. This will increase your product sales.
